internal/gtkcord/sidebar/guilds/folder.go

```diff
@@ -158,6 +157,22 @@ func (f *Folder) Set(folder *gateway.GuildFolder) {
 	for _, g := range f.Guilds {
 		g.Invalidate()
 	}
+
+	// After guilds are loaded, read their labels and set the folder name if unset.
+	if len(folder.Name) == 0 {
```

Comment:
You should do `folder.Name == ""` here, as it's more consistent with the rest of the code.